tech-specs/src/explore/design/intent_gossip/topic.md

```diff
@@ -0,0 +1,13 @@
+# Topic
+
+A topic is string and a encoding that describes this sub-network. In a topic all
+intents must use the exact same encoding. That encoding is known by matchmakers
+so it can match them.  Whenever a node subscribes to a new topic it informs all
+connected nodes. Each of them tries to subscribe to it and continue the
```

Comment:
Other nodes automatically also subscribe? Shouldn't they opt-in?

Reply:
A node can express an topic filter to prevent subscription to undesired topic. This filter is composed of a whitelist, a regex and a maximum subscription. 

This I describe in the next paragraph, I'm going to reformulate this sentence to express that